tests/benchmarks/multicore/idle_outside_of_main/sysbuild.cmake

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+  message(FATAL_ERROR "REMOTE_BOARD must be set to a valid board name")
+endif()
+
+# Add remote project
+ExternalZephyrProject_Add(
+    APPLICATION remote
+    SOURCE_DIR ${APP_DIR}/remote
+    BOARD ${SB_CONFIG_REMOTE_BOARD}
+    BOARD_REVISION ${BOARD_REVISION}
+  )
+set_property(GLOBAL APPEND PROPERTY PM_DOMAINS CPUNET)
```

Comment:
What this property is used for? I see that this code is in idle, idle_spim, silde_twim and idle_uarte benchmarks.
Search doesn't show any use of PM_DOMAINS symbol.

Reply:
It was there at idle from beginning - https://github.com/nrfconnect/sdk-nrf/commit/729226d538f4ee91f04ad0a707ae09341b905ac4
I just copied it.